types/next-pwa/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "webworker"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
```suggestion
        "forceConsistentCasingInFileNames": true
```
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

Reply:
Disabling the `esModuleInterop` flag causes the following issues (refer to print).
 
![image](https://user-images.githubusercontent.com/90518458/205439821-f21df0c6-3095-4228-a263-4613f72d8ad9.png)

Is there a known workaround to this? As of yet, I am only aware of resolving this issue by enabling the flag. Would love to know if there is a solution!

Reply:
there is no 'workaround' here required, JS module is CJS:
> refer to print

Please copy test you're pointing to next time, this is just an image. Do you get this error when testing? This module is a CJS. I'll try locally

Reply:
so the errors is due to `next` shipping ESM type of declaration files only, so when referenced here, spills the errors.
One should describe why changing default config (which is why 'check config' label was added) in first place, when starting PR.

Reply:
> One should describe why changing default config (which is why 'check config' label was added) in first place, when starting PR.

My bad! Before opening the PR the `tsconfig.json` had comments right above the `esModuleInterop` describing why it was enabled, which is not the default config. Forgot to add a comment to the **PR** after removing the `tsconfig.json` comment to comply to DT's package rules.